packages/react-data-grid/style/rdg-header.less

```diff
@@ -22,6 +22,13 @@
   width: 10px;
 }
 
+@supports (position: sticky) or (position: -webkit-sticky) {
```

Comment:
Do we need this? Currently we render all the header cells and adjust the scrolleft on scroll

Reply:
We also iterate through all the columns to set/unset the scroll position of each header row cells.

I figure we should optimize this for browsers that support sticky position, no?